libcodechecker/libhandlers/store.py

```diff
@@ -107,6 +107,13 @@ def add_arguments_to_parser(parser):
                              "the '--name' parameter given to 'codechecker-"
                              "analyze' will be used, if exists.")
 
+    parser.add_argument('--tag',
```

Comment:
A new argument is added, but the documentation for the `store` command was not updated.